yarn.lock

```diff
@@ -1620,7 +1663,18 @@
     "@dhis2/prop-types" "^1.6.4"
     classnames "^2.3.1"
 
-"@dhis2/ui@^6.5.3", "@dhis2/ui@^6.9.0":
+"@dhis2/ui@^6.5.3":
```

Comment:
You can run dedupe (`npx yarn-deduplicate yarn.lock`) to remove the duplicate `ui` versions (see `@dhis2/ui@^6.9.0` below).

Reply:
Done.